src/createAsyncMiddleware.js

```diff
@@ -1,39 +1,59 @@
-const promiseToCallback = require('promise-to-callback')
+/**
+ * JsonRpcEngine only accepts synchronous middlewares.
```

Comment:
Same notes here as in the README; "middleware" is already plural, and callback-based middleware is still asynchronous.

Reply:
1448044